ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6556,6 +6556,24 @@ RFVSC2X:
   implements:
   - OPERATIONAL
 
+DSTLM:
```

Comment:
Suggest breaking this into two types (temp and level monitoring should be separated) re-naming to follow precedent. For example you can use the existing `LM` type and create a new one similar to it like `LM2X`. For temp monitoring, see `HWTTC` for an example.